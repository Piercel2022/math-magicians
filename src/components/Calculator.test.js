import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from './calculator';

it('Calculator component is working', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
