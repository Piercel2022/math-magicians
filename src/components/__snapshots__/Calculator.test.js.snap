// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component is working 1`] = `
<div
  className="calculatorSection"
>
  <h1>
    Let's do some math!
  </h1>
  <div
    className="calc"
  >
    <p
      className="result"
    >
      <span>
        0
      </span>
      <span />
      <span />
    </p>
    <div
      className="calculator"
    >
      <button
        className="grayButton"
        name="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="grayButton"
        name="+/-"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="grayButton"
        name="%"
        onClick={[Function]}
        type="button"
      >
        % 
      </button>
      <button
        className="orangeButton"
        name="รท"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="grayButton"
        name="7"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="grayButton"
        name="8"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="grayButton"
        name="9"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="orangeButton"
        name="x"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="grayButton"
        name="4"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="grayButton"
        name="5"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="grayButton"
        name="6"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="orangeButton"
        name="-"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="grayButton"
        name="1"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="grayButton"
        name="2"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="grayButton"
        name="3"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="orangeButton"
        name="+"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="grayButton-0"
        name="0"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="grayButton"
        name="."
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="orangeButton"
        name="="
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
